### DISCUSSION TITLE
Welcome to easy-switcher Discussions!

### DISCUSSION THREAD

**Discussion by freemind001** [freemind001/easy-switcher#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qua-rus** [comment]
Здравствуйте.
Спасибо за приложение, давно ищу.
Если будет желание и возможность - опубликуйте описание на Хабре, думаю - многие будут рады вашему приложению для Linux.

**Reply by freemind001** [reply]
Благодарю за отзыв
